src/tools/sfdk/sfdkconstants.h

```diff
@@ -40,5 +40,10 @@ const char EXIT_ABNORMAL_ENV_VAR[] = "SFDK_EXIT_ABNORMAL";
 const int EXIT_ABNORMAL_DEFAULT_CODE = 120;
 const char NO_QPA_PLATFORM_MINIMAL_ENV_VAR[] = "SFDK_NO_QPA_PLATFORM_MINIMAL";
 
+const char OS_VARIANT[] = "%{OsVariant}";
+const char OS_VARIANT_NOSPACE[] = "%{OsVariantNoSpace}";
+const char OS_VARIANT_UNDERLINE[] = "%{OsVariantUnderline}";
+const char IDE_VARIANT[] = "%{IdeVariant}";
+const char SDK_VARIANT[] = "%{SdkVariant}";
```

Comment:
- The variable names are a bit misleading, I would suffix them with "_MACRO", e.g., OS_VARIANT_MACRO.
- "OsVariant:NoSpace" and "OsVariant:Underline" would be better (notice the colon).
- I would prefer moving these to dispatch.cpp, there is already a block with constants at the beginning of that file

Reply:
Updated